APNGKit/APNGImageView.swift

```diff
@@ -150,7 +158,7 @@ open class APNGImageView: APNGView {
         
         addObservers()
```

Comment:
It is unnecessary code when both autoStopAnimationBeforeResignActive and autoStartAnimationAfterResumeActive are false.
How about it?
`private var isObserverNeeded: Bool {
        autoStopAnimationBeforeResignActive || autoStartAnimationAfterResumeActive
}`
`if isObserverNeeded {
    addObservers()
}`

Reply:
@Byounggon `addObservers` is at initializer, and also the required `init(coder:)` and `init(frame:)` is difficult to have default value of `autoStopAnimationBeforeResignActive` `autoStartAnimationAfterResumeActive` parameters before initialization